Dockerfile

```diff
@@ -1,14 +1,14 @@
-# Build the manager binary
-FROM golang:1.17 as builder
-
+# Build the operator binary
+FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 as builder
```

Comment:
@guicassolato perhaps you want to keep the `1.17` floating tag here to always build on latest / most secure 1.17 image?

Reply:
I'm afraid there ain't one 😕

Reply:
<img width="1153" alt="Screenshot 2022-07-27 at 10 57 58 AM" src="https://user-images.githubusercontent.com/2420882/181193796-9a10b0e0-488f-4677-8eb0-61b0968ec0d7.png">

?

Reply:
Sadly, that one is rhel, not UBI.

<img width="1728" alt="Screenshot 2022-07-27 at 10 03 06" src="https://user-images.githubusercontent.com/1842261/181194750-169ee874-ce84-4607-8a66-f7349f461ced.png">

Reply:
@guicassolato nice catch. 

ok - let's proceed with merging and we fix later, in case the tag becomes available. (I've cc'ed you on an email about that)

Reply:
Thanks, @gsaslis!